lib/brakeman/parsers/rails3_erubis.rb

```diff
@@ -1,45 +1,74 @@
 Brakeman.load_brakeman_dependency 'erubis'
 
-#This is from Rails 3 version of the Erubis handler
+# This is from Rails 5 version of the Erubis handler
+# https://github.com/rails/rails/blob/ec608107801b1e505db03ba76bae4a326a5804ca/actionview/lib/action_view/template/handlers/erb.rb#L7-L73
 class Brakeman::Rails3Erubis < ::Erubis::Eruby
 
   def add_preamble(src)
-    # src << "_buf = ActionView::SafeBuffer.new;\n"
+    @newline_pending = 0
+    src << "@output_buffer = output_buffer || ActionView::OutputBuffer.new;"
```

Comment:
Just as a note, this line and the postamble will be ignored anyway. Otherwise this causes a lot of overhead processing templates.